### DISCUSSION TITLE
In last two days getting HUGE eth hash rate decrease with chia v 1.13 or 1.14 running concurrently with mining eth

### DISCUSSION THREAD

**Discussion by mwinschel** [Chia-Network/chia-blockchain#3972] [Support]
For some reason in the last day or so I am getting a halving of my eth hashrate on my 3080 from mid to high 90s down into 30s to maybe 40.  I was puzzled as to what was causing it and it appears to just be the base Chia program operation.  It appears to be pulling 20% usage on my GPU just running it without any plotting operations all of a sudden.  not sure what the cause is and could use some suggestions.  Is it possible that farming the few plots i have made is causing the issue?  I have not been using chia long but for the first few days there was no effect on mining ETH at all.

**Comment by sargonas** [comment]
Most consumer motherboards have shared bus lanes between pciex and m2 slots. Quite often, the one of your m2 shares with you x16, and somtimes SATA may as well. Check your MB documentation specs to find out because it's quite likely that if you are pushing an M2 to the max, AND pushing a GPU to the max, they are fighting for the same bus lanes and are restricting each other in a tug of war battle.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
